iModelCore/BeSQLite/BeSQLite.cpp

```diff
@@ -3148,10 +3148,31 @@ DbResult Db::TruncateTable(Utf8CP tableName) const
 /*---------------------------------------------------------------------------------**//**
 * @bsimethod
 +---------------+---------------+---------------+---------------+---------------+------*/
-bool Db::TableExists(Utf8CP tableName) const
+bool Db::TableExists(Utf8CP tableName, Utf8CP tableSpace) const 

```

Comment:
Since you just parse tablespace from tableName and the function previously expected tablename to be optionally qualified by tablespace. So we can avoid adding this argument and just depend on caller to give us e.g. `TableExists('TEMP.table1')` instead of `TableExists('table1', 'TEMP')`